### ISSUE TITLE
Bottom cooling mechanism in Amundsen etc in UP

### ISSUE THREAD

**Issue by adele157** [adele157/easterlies-collaborative-project#25]
There is a [strong bottom cooling and freshening signal on the shelf](https://github.com/adele157/easterlies-collaborative-project/issues/16#issuecomment-884591714) in regions such as the Amundsen, Totten and Vincennes Bay in the UP simulation. This is consistent with the expected Ekman pumping response. However, it is confusing, because the cooling only shows up below ~200m depth, and above that there is warming/salinification. The isopycnals shallow in the upper ~200m and deepen below 200m. 
![amundsen](https://user-images.githubusercontent.com/8506963/127412994-f51db484-3946-4f29-a9c6-f1d2492c72dc.jpg)

As suggested by @AndyHoggANU, perhaps we can explain these changes as follows:
1. The Ekman pumping response (cooling/freshening/isopycnals moving down in UP) is communicated throughout the water column and this is what we are seeing at depth in these regions.
2. However, at the surface this Ekman pumping response is being overwhelmed by something else (maybe more sea ice production driving increased salinity?).

To investigate this idea further, we are going to:
- Look at [surface buoyancy fluxes](https://github.com/adele157/easterlies-collaborative-project/issues/22), 
- Plot seasonal transects in these regions (the hypothesis is that the Ekman pumping response should be seen all year round, but the surface warming/salinification may be more seasonal if it is tied to sea ice production changes),
- Make animations of transects in these regions to see if the two responses outlined above have different timescales when they appear and can therefore be separated,
- Assess the plots of [wind stress](https://github.com/adele157/easterlies-collaborative-project/issues/15) and [Ekman pumping](https://github.com/adele157/easterlies-collaborative-project/issues/20) in these regions to see if they are consistent with the Ekman pumping response we think we are seeing at depth here (i.e. more Ekman pumping in UP in these regions).

**Comment by adele157** [comment]
I don't think the seasonal anomaly transects are very enlightening. The isopycnals are still moving upwards in the upper 200m in the UP simulation (i.e. opposite to what we expect from Ekman pumping) throughout nearly the whole year (with the exception of Autumn in Amundsen below). It seems to be the salinity anomaly driving the upper ocean change. I wonder if the increased surface salinity is due to the DSW increase and this is being advected around the whole continent? So maybe it's not a local change in the upper ocean as Andy proposed above (which we might expect to be more seasonal), but rather a remotely driven salinification which remains throughout the year. But if you look at the [surface salinity pattern here](https://user-images.githubusercontent.com/8506963/124698521-5dc9e680-df2c-11eb-8402-6e76e4ecc880.png), it does seem to be very coastally intensified in particular regions far from DSW formation sites.

Here are some seasonal anomalies for the UP simulation for the Amundsen and Totten (dashed lines are control isopycnals):

Amundsen:
![AmundsenW_seasons_UP](https://user-images.githubusercontent.com/8506963/127565057-9e1d0852-e051-4010-8f64-780afc432737.png)

Totten:
![Totten_seasons_UP](https://user-images.githubusercontent.com/8506963/127565107-b57dacea-6d5f-46be-b163-7a50567ef2c0.png)

**Comment by matthew-england-unsw** [comment]
Interesting thanks Adele.  I wonder about looking at some animations of SSS, SST, sea-ice and even along these meridional transects to check the transient evolution?   If the DSW adjustments are the source of the surface signatures later on, we should see those regions emerge with the initial T-S anomalies, and then advect around the continent.  Be great to see how the anomalies evolve and where the initial anomalies originate from.

**Comment by PaulSpence** [comment]
Further analysis: mld, surface heat flux, age (cdw is old)?, cdw, split forcings with cross sections of temp, sea ice in split forcings, along shore flow changes?

**Comment by wghuneke** [comment]
Going through (part of) the list suggested by Paul above:

MLD: We have an issue [here](https://github.com/adele157/easterlies-collaborative-project/issues/14) which shows that the mixed layer **deepens** in the Amundsen Sea. We don't have the split for zonal/merid, but I'd expect increased Ekman would give us this response? (Or increased DSW formation, but that's not too likely in the Amundsen Sea.)

Surface heat flux: A map of the surface heat flux is [here](https://github.com/adele157/easterlies-collaborative-project/issues/22), nothing crazy happening in the Amundsen Sea.

Cross sections: I replotted the two transects in the Amundsen Sea shown in [this issue](https://github.com/adele157/easterlies-collaborative-project/issues/16) for temp, salt, age and zonal velocity and added the zonal and meridional cases. Dashed contours are from the control case, solid contours are from the respective experiment.

Eastern transect: (PIG)
- overall cooling in WIND+ with max in the subsurface (matches deepening of mixed layer)
- inner shelf cooling dominated by zonal component, 'outer' shelf cooling by meridional component
- split between the inner and outer shelf also in salinity and age 
- slope current eastward at this location and increases in both components -> do we get less transport of CDW onto the shelf and hence the young, cold and fresh anomaly at outer shelf?
<img width="1004" alt="AmundsenE_temp" src="https://user-images.githubusercontent.com/15355753/160039510-c19e2516-fdbe-4f17-9ab7-1ba50583bb7c.png">
<img width="1005" alt="AmundsenE_salt" src="https://user-images.githubusercontent.com/15355753/160039518-cefcaa80-3478-4416-a2f2-d11dea01a8ee.png">
<img width="1005" alt="AmundsenE_age" src="https://user-images.githubusercontent.com/15355753/160039530-7eb7ad50-1105-4d9e-bf22-5333e6882301.png">
<img width="1007" alt="AmundsenE_vel" src="https://user-images.githubusercontent.com/15355753/160039536-ad9872a9-8930-49e8-bcf9-4e9be3572454.png">


Western transect:
- cooling strongest at depth (on shelf) and at the subsurface (off shelf)
- pretty much all the cooling from meridional component (shelf not as wide as further west)
- salinity increases near the surface and decreases at depth -> hint that there is more convection?
- slope current (now westward) weakens
<img width="1055" alt="AmundsenW_temp" src="https://user-images.githubusercontent.com/15355753/160039549-5e7da304-aa50-4a0e-85e8-24fe3f772b18.png">
<img width="610" alt="AmundsenW_salt" src="https://user-images.githubusercontent.com/15355753/160039553-02203093-8176-42ed-9248-a440531dc8bd.png">
<img width="610" alt="AmundsenW_age" src="https://user-images.githubusercontent.com/15355753/160039559-dec2474f-aefe-43cb-b936-1c2685b03bb1.png">
<img width="1005" alt="AmundsenW_vel" src="https://user-images.githubusercontent.com/15355753/160039566-4d625fc6-3a61-43b7-a466-4a023fb6e490.png">

**Comment by adele157** [comment]
Here's some mixed layer depth plots from the Amundsen region.

First in units of m, showing mixed layers up to about 300m deep, and these deepen by 50-100m in the meridional perturbation.
<img width="1181" alt="Screen Shot 2022-04-06 at 1 58 37 pm" src="https://user-images.githubusercontent.com/8506963/161893172-05458ef7-6a6a-44e5-a712-2e7dbf2083f6.png">

And then as a fraction of ocean depth:
<img width="1181" alt="Screen Shot 2022-04-06 at 1 58 48 pm" src="https://user-images.githubusercontent.com/8506963/161893451-fa0b6d51-5208-478a-91b8-665e147b4310.png">

I think these could be deep enough to be causing the changes we're seeing in temperature on the shelf.

**Comment by matthew-england-unsw** [comment]
This is great, thanks for chasing up these MLD analyses Adele.   I agree that mixed layer depth variations of this magnitude are likely enough to give us those temperature anomalies at depth.  They would evolve over time and mature to the signals we see.  And this also corroborates the temperature anomaly sections that Wilma drew up above — with maximum temperature anomalies at or just below the base of the mixed layer, if I have read it correctly.

**Comment by adele157** [comment]
Always useful to see too how much you miss by looking at monthly averages of MLD (left panels above). The deep events only show up on daily timescales (right panels).

**Comment by matthew-england-unsw** [comment]
Yes that’s impressive — proper sporadic convection happening on daily time-scales.  I guess as weather systems move across the domain, occasionally bringing particularly conducive DSW formation conditions.  Nice!   What a model.
